etna/lib/commands.rb

```diff
@@ -339,7 +339,7 @@ def execute(project_name, model_name, file_path, execute: false)
             request = Etna::Clients::Magma::RetrievalRequest.new(project_name: project_name)
             request.model_name = model_name
             request.attribute_names = 'all'
-            request.record_names = 'all'
+            request.record_names = []
```

Comment:
Basically `all` here was not necessary and caused performance issues for models with many records.